client/components/domains/domain-search-results/index.jsx

```diff
@@ -175,7 +176,11 @@ class DomainSearchResults extends React.Component {
 		}
 
 		return (
-			<div className="domain-search-results__domain-availability">
+			<div
+				className="domain-search-results__domain-availability"
+				aria-live="polite"
```

Comment:
This doesn't seem to do anything in my VoiceOver testing – I think because this entire element is added/changed at once, rather than changing content inside of it.

Reply:
I've removed this for now on 4a95bfb315198bfecd62eabfac3526e36e9a49a8 because it doesn't work entirely as expected. To make it work, I'll have to reorder some elements and change the way some of the components render so they update content as `aria-live` expects. I would like to follow that up on a new PR so this don't became stale.